filters/auth/grantflowstate.go

```diff
@@ -0,0 +1,132 @@
+package auth
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math/rand"
```

Comment:
this should be crypto/rand, not?

Reply:
Removed as a result of removing the `randString()` function